src/axom/quest/ScatteredInterpolation.hpp

```diff
@@ -317,6 +428,45 @@ class ScatteredInterpolation
     }
   }
 
+  /**
+   * \brief Given a location in space, find the associated indices
+   * and interpolation weights with respect to the input mesh points
+   *
+   * \param [in]  query_pt The point at which we want to interpolate
+   * \param [out] indices The indices of the points from the input mesh in the support of \a query_pt
+   * \param [out] weights The interpolation weights associated with each input point in \a indices
+   *
+   * \returns true if \a query_pt is found within a cell of the Delaunay complex; false otherwise.
+   *  If true, the associated indices from points in the input mesh are returned in \a indices
+   *  and the interpolation weights for each point are returned in \a weights
+   */
+  bool getInterpolationWeights(const PointType& query_pt,
```

Comment:
@dtaller -- here is the function you requested. Given a point, it return the indices of the points in the support of that point and their associated interpolation weights.
I use it in the `scattered_interpolation` example as a second way of interpolating the fields and to interpolate positions.

Reply:
Thanks! I appreciate it!